src/main/flight/mixer.c

```diff
@@ -150,6 +150,7 @@ static const motorMixer_t mixerTricopter[] = {
     { 1.0f,  1.0f, -0.666667f,  0.0f },     // LEFT
 };
 
+#ifndef DISABLE_UNCOMMON_MIXERS
```

Comment:
prefer USE instead of DISABLE - postive rather than negative.  e.g.  `USE_UNCOMMON_MIXERS`